chroma_core/lib/service_config.py

```diff
@@ -193,10 +194,12 @@ def _setup_ntp(self, server):
                                                                                    error))
             raise RuntimeError("Failure when writing ntp config: %s" % error)
 
-        error = firewall_control.add_rule("123", "udp", "ntpd")
-        if error:
-            log.error("firewall command failed:\n%s" % error)
-            raise RuntimeError("Failure when opening port in firewall for ntpd: %s" % error)
+        if ServiceControl.create('firewalld').running:
```

Comment:
Did we decide to take out firewalld?

Reply:
We decided to not install the dep for docker, as it's not needed there. We still need to support it for non-docker installs.